libraries/tentacle.rb

```diff
@@ -41,10 +43,31 @@ def installer_url(version)
       "https://download.octopusdeploy.com/octopus/Octopus.Tentacle.#{version}-x64.msi"
     end
 
-    def resolve_port(polling, port)
+    def tentacle_exists?(server, api_key, thumbprint)
+      ::JSON.parse(api_client(server, api_key).get("/machines/all?thumbprint=#{thumbprint}")).any? do |machine|
```

Comment:
@spuder Went ahead and change it to use the endpoint that exists.  Tested and it did work so I think this should solve the high usage of notes checking in.